src/protocols/rdp/Makefile.am

```diff
@@ -268,6 +268,7 @@ rdp_keymaps =                   \
     $(srcdir)/keymaps/en_gb_qwerty.keymap \
     $(srcdir)/keymaps/en_us_qwerty.keymap \
     $(srcdir)/keymaps/es_es_qwerty.keymap \
+    $(srcdir)/keymaps/latam_qwerty.keymap \
```

Comment:
A couple of things, here:
- Is "latam" actually the proper identifier for this keyboard layout?  I would suspect something following existing patterns - like "es_la_qwerty" or the like?  I don't know if there's an official standard for this or just what we're calling it in Guacamole, but it should be consistent.
- It should also be organized alphabetically - so if "es_la_qwerty" is the correct name for the layout, this is the correct spot - if "latam_qwerty" is correct, then it should be down a little further in the file.

Reply:
https://askubuntu.com/questions/8797/what-is-the-two-character-language-code-for-the-latin-american-keyboard-layout

Reply:
Thanks.  I'm good with this as long as @mike-jumper or anyone else doesn't have any objections?

Reply:
I agree that we should prefer to remain consistent with the existing convention (`[language code]-[country code]-[variant]`) or, if that isn't working any longer in practice, investigate whether we should be using a different convention.

Following current convention, if "latam" is the proper identifier for all countries associated with this keyboard layout, it sounds like the layout identifier that follows our convention would be `es-latam-qwerty`.

**BUT**, from the [forum thread linked above](https://askubuntu.com/questions/8797/what-is-the-two-character-language-code-for-the-latin-american-keyboard-layout), we may want to look into a better system:

> ... That aside, I would recommend against using language codes to indicate keyboard layouts. There is a separate system already in place for abbreviating the names of the layouts themselves. See `shortDescription` in [xkb-data](http://packages.ubuntu.com/maverick/xkb-data)'s `base.xml`.